.github/workflows/labels-verifier.yaml

```diff
@@ -13,9 +13,13 @@ jobs:
           github.event.pull_request.merged == true &&
           ! contains(join(github.event.pull_request.labels.*.name, ', '), 'kind/') &&
           ! contains(github.event.pull_request.labels.*.name, 'merge')
-        uses: actions/github-script@0.3.0
+        uses: actions/github-script@v3
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           script: |
-            const { issue: { number: issue_number }, repo: { owner, repo }  } = context;
-            github.issues.createComment({ issue_number, owner, repo, body: 'This PR was merged without a "kind/" label, please correct.' })
+            github.issues.createComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              body: 'This PR was merged without a "kind/" label, please correct.',
+            })
```

Comment:
to be fair 2 lines solution looked more concise. But we can use this one for consistency

Reply:
Inarguably more concise, but I think this is more readable :)

Reply:
Indeed I went for readability, especially since the order of arguments here was different from the one on https://octokit.github.io/rest.js/v18#issues-create-comment , so I think specifying the argument names is also safer.